src/parser/jobs/brd/modules/Util.js

```diff
@@ -31,7 +31,12 @@ export default class Util extends Module {
 	}
 
 	getDowntimeLength(timestamp) {
-		const window = this.downtime.getDowntimeWindows().filter(x => x.start <= timestamp && x.end >= timestamp)
+		const window = this.downtime.getDowntimeWindows()
+			.map(window => ({
+				start: this.parser.epochToFflogs(window.start),
+				end: this.parser.epochToFflogs(window.end),
+			}))
+			.find(x => x.start <= timestamp && x.end >= timestamp)
```

Comment:
This was trying to access the result of `filter` as though it were a single item. Assuming bug, fixing with filter -> find.